src/core/seat/keyboard.cpp

```diff
@@ -121,14 +121,18 @@ void wf::keyboard_t::setup_listeners()
 wf::keyboard_t::keyboard_t(wlr_input_device *dev) :
     handle(wlr_keyboard_from_input_device(dev)), device(dev)
 {
-    model.load_option("input/xkb_model");
-    variant.load_option("input/xkb_variant");
-    layout.load_option("input/xkb_layout");
-    options.load_option("input/xkb_options");
-    rules.load_option("input/xkb_rules");
-
-    repeat_rate.load_option("input/kb_repeat_rate");
-    repeat_delay.load_option("input/kb_repeat_delay");
+    auto section =
```

Comment:
Seems like we'd want the same for pointer? https://github.com/WayfireWM/wayfire/blob/56532427bea34ed9d777708b52356220c366de73/src/core/seat/pointing-device.cpp#L12-L31

Reply:
I was thinking what other bits we might want this for.. adding for pointer makes sense to me, will update.